api/relations/v0/lookup.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package kessel.relations.v0;
+
+import "google/api/annotations.proto";
+import "relations/v0/common.proto";
+
+option go_package = "ciam-rebac/api/relations/v0;v0";
+option java_multiple_files = true;
+option java_package = "org.project_kessel.api.relations.v0";
+
+service KesselLookupService {
+    rpc LookupSubjects(LookupSubjectsRequest) returns (stream LookupSubjectsResponse) {
+        option (google.api.http) = {
+            get: "/v0/subjects"
+        };
+    };
+}
+
+message LookupSubjectsRequest {
+    ObjectType subject_type = 1;
+    optional string subject_relation = 2;
+    string relation = 3;
+    ObjectReference resource = 4;
```

Comment:
From Raj & I: could we flip the order of resource and subject related fields? (e.g. resource, relation, subject type, subject relation)